lib/gds_api/test_helpers/router.rb

```diff
@@ -64,6 +88,11 @@ def stub_router_commit
 
     private
 
+      def stub_get_route(path)
+        stub_http_request(:get, "#{ROUTER_API_ENDPOINT}/routes")
+          .with(query: { "incoming_path" => path })
```

Comment:
Is it worth stubbing this with the bearer token in so we know that anything using it also passes that along?

Reply:
Yep, I think that's a good idea, I've added that now along with removing `Plek.current`!